src/App.js

```diff
@@ -41,10 +50,32 @@ const App = () => {
               path="MyCohort"
               element={
                 <ProtectedRoute>
+                  {/* <Card children="child"/> */}
+                  {/* <Button text="hello"/> */}
+                  {/* <Comment name="jimbo" content="hello" /> */}
+                  {/* <CredentialsCard/> */}
+                  {/* <SocialLinks/> */}
+                  {/* <Header/> */}
```

Comment:
It looks like these can be removed

Reply:
Removed as requested - though "please note they exit on main branch"